website/content/docs/agent/index.mdx

```diff
@@ -366,6 +366,63 @@ service {
 </Tab>
 </Tabs>
 
+## Client Node with Multiple Interfaces or IP addresses
+
+The following example shows how to configure Consul to listen on multiple interfaces or IP addresses using a [go-sockaddr] template.
+
+The `bind_addr` is used for internal RPC and Serf communication (see the Agent Configuration for [bind_addr](/docs/agent/options#bind_addr) for more information).
+
+The `client_addr` configuration specifies IP addresses used for HTTP, HTTPS, DNS and GRPC servers. (see the Agent Configuration for [client_addr](/docs/agent/options#client_addr) for more information).
+
+<Tabs>
+<Tab heading="HCL">
+
+```hcl
+node_name = "consul-server"
+server    = true
+bootstrap = true
+ui_config {
+  enabled = true
+}
+datacenter = "dc1"
+data_dir   = "consul/data"
+log_level  = "INFO"
+
+# used for internal RPC and Serf
+bind_addr = "0.0.0.0"
+
+# Used for HTTP, HTTPS, DNS, and GRPC addresses.
+# loopback is not included in GetPrivateInterfaces. More information can be found in the go-sockaddr documentation 
```

Comment:
could we link specifically which documentation folks can look at here?

Reply:
the description in go-sockaddr doesn't have an anchor. I can include the reasoning here?